src/quo2/components/list_items/preview_list/view.cljs

```diff
@@ -0,0 +1,150 @@
+(ns quo2.components.list-items.preview-list.view
+  (:require [quo2.components.avatars.account-avatar.view :as account-avatar]
+            [quo2.components.avatars.user-avatar.view :as user-avatar]
+            [quo2.components.icon :as quo2.icons]
+            [quo2.components.markdown.text :as quo2.text]
+            [quo2.foundations.colors :as colors]
+            [quo2.theme :as quo.theme]
+            [quo2.components.list-items.preview-list.properties :as properties]
+            [react-native.core :as rn]
+            [react-native.fast-image :as fast-image]
+            [react-native.hole-view :as hole-view]))
+
+;; This overflow item needs to be cleaned up once the "number tag" component is implemented
+;; https://github.com/status-im/status-mobile/issues/17045
+(defn get-overflow-color
+  [blur? blur-light-color blur-dark-color light-color dark-color theme]
+  (if blur?
+    (colors/theme-colors blur-light-color blur-dark-color theme)
+    (colors/theme-colors light-color dark-color theme)))
+
+(def more-icon-for-sizes #{16 14})
+
+(defn overflow-label
+  [{:keys [label size blur? border-radius margin-left theme more-than-99-label]}]
+  [rn/view
+   {:style {:width            size
+            :height           size
+            :margin-left      margin-left
+            :border-radius    border-radius
+            :justify-content  :center
+            :align-items      :center
+            :background-color (get-overflow-color
+                               blur?
+                               colors/neutral-80-opa-5
+                               colors/white-opa-5
+                               colors/neutral-20
+                               colors/neutral-90
+                               theme)}}
```

Comment:
I think these kind of style maps should be in the `style` namespace

Reply:
This `overview-label` needs to be replaced with a `number-tag` component. This will be cleaned up after its implementation.

https://github.com/status-im/status-mobile/pull/17051/files#diff-7e03eab94812b8180e2654d500554953d15c886a8aec4d2d1b8dc0506f3646f9R13-R14

https://github.com/status-im/status-mobile/issues/17045